requirements-dev.txt

```diff
@@ -0,0 +1 @@
+bumpversion
```

Comment:
Should this not include the requirements-test? So that the tools are available to a developer to run.